amplipi/streams.py

```diff
@@ -1165,6 +1166,23 @@ def connect(self, src):
     src_config_folder = f"{utils.get_folder('config')}/srcs/{src}"
     os.system(f'mkdir -p {src_config_folder}')
 
+    # HACK check if url is a playlist and if it is get the first url and play it
```

Comment:
I don't love it, but it wasn't made to be loved. i like not pulling in a new dep, and also your tests give me some confidence in this.